openedx/core/djangoapps/verified_track_content/partition_scheme.py

```diff
@@ -0,0 +1,146 @@
+"""
+UserPartitionScheme for enrollment tracks.
+"""
+from courseware.masquerade import (  # pylint: disable=import-error
+    get_course_masquerade,
+    get_masquerading_group_info,
+    is_masquerading_as_specific_student,
+)
+from course_modes.models import CourseMode, EnrollmentMode
+from student.models import CourseEnrollment
+from opaque_keys.edx.keys import CourseKey
+from openedx.core.djangoapps.verified_track_content.models import VerifiedTrackCohortedCourse
+from xmodule.partitions.partitions import NoSuchUserPartitionGroupError, Group, UserPartition
+
+
+# These IDs are guaranteed to not overlap with Groups in the CohortUserPartition or the RandomUserPartitionScheme
```

Comment:
Christina: change the comment to note that enrollment_mode.id will be low integers, under 100.